algosdk/v2client/algod.py

```diff
@@ -125,7 +125,11 @@ def application_info(self, application_id, **kwargs):
             application_id (int): The ID of the application to look up.
         """
         req = "/applications/" + str(application_id)
-        return self.algod_request("GET", req, **kwargs)
+        resp = self.algod_request("GET", req, **kwargs)
```

Comment:
Adding `application-account` to **app-info** response as typically need this for funding apps (a requirement for inner app calls).